### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by IndrashisChatterjee**

### CODE REVIEW

**Comment by IndrashisChatterjee** [comment]
Please help me by Accepting and merging

File: Dev_Jokes_and_Quotes/README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@
 
 ### ALL THE BEST :heart:
 
-
+***
+Think twice code Once !!
```
Comment (ritiksoni00) [inline Dev_Jokes_and_Quotes/README.md:15]: do something extra work dude! if you just wanna grab a tee you can add JOkes in-jokes folder easy

Comment (IndrashisChatterjee) [reply Dev_Jokes_and_Quotes/README.md:15]: Was it not only adding to readme ? Okay i will post in jokes but please help me by confirming

Comment (ritiksoni00) [reply Dev_Jokes_and_Quotes/README.md:15]: add jokes leave this one!

**Pull request closed without merging**